src/components/bar.cpp

```diff
@@ -458,7 +458,16 @@ void bar::restack_window() {
 
   auto restacked = false;
 
-  if (wm_restack == "bspwm") {
+  if (wm_restack == "generic") {
+    auto children = m_connection.query_tree(m_connection.screen()->root).children();
+    auto lowest = *children.begin();
+    if (lowest != m_opts.window) {
+      const unsigned int value_mask = XCB_CONFIG_WINDOW_SIBLING | XCB_CONFIG_WINDOW_STACK_MODE;
+      const unsigned int value_list[2]{lowest, XCB_STACK_MODE_BELOW};
+      m_connection.configure_window_checked(m_opts.window, value_mask, value_list);
```

Comment:
This can throw an exception which would crash the bar. Please add a try-catch block that prints an error (and doesn't set `restacked`)